README.md

```diff
@@ -1 +1,57 @@
-# E-commerce-app-team0
+
+# E-commerce-app-Team0
+
+### Why we did this? 
+- This app was designed and implemented to make people's lifes easier selling and buying things online. 
+- This is the final project from the Fundamentals program in web development provided by Gaza Sky Geeks (GSG). 
+- It aims at developing our web development skills and gaining new skills other than those acquired during the program. 
+---
+### What is this app?
+- It is a platform for selling and buying products
+---
```

Comment:
@hMZhbhr Hi! It might be helpful to include a `### How to run this app` section here, with instructions on cloning and running the app locally.  It will probably be very simple instructions for this app, but it's a good habit to add a section like this for all app readme files.

Reply:
It is, but by the end of it, we are going to add github pages link, What do you think?

Reply:
I think they are a little different and both would be good!
The github pages link will tell people where they can go and try the latest "published" version of the app (without needing to clone the repo and run it locally)
and the `### How to run this app` section will tell other developers what to do to start developing the app on their local machine.

Reply:
Ok, I'll do it, but in a separate pull request since I merged this one.

Reply:
ok sounds good, thanks!